source/fuzz/transformation_access_chain.cpp

```diff
@@ -22,21 +22,82 @@
 namespace spvtools {
 namespace fuzz {
 
+namespace {
+// Given a context, a value and a type id, returns the id of an integer
+// constant with the same type and given value.
+// The type id must correspond to an integer type.
+uint32_t FindIntConstant(opt::IRContext* ir_context, uint32_t value,
+                         uint32_t int_type_id) {
+  auto int_type_inst = ir_context->get_def_use_mgr()->GetDef(int_type_id);
+
+  assert(int_type_inst && "The given type id must exist.");
+
+  auto int_type = ir_context->get_type_mgr()
+                      ->GetType(int_type_inst->result_id())
+                      ->AsInteger();
+
+  assert(int_type && "The given type id must correspond to an integer type.");
+
```

Comment:
Could put this in `fuzzer_util.cpp`? If so, change the name of `bound_minus_one`. Documentation comment should start with "Returns...", "Finds...", or similar. Make sure you also document the assertions; i.e. the "requirements" of this function. E.g.

Returns the id of an integer constant with type |int_type_id| and value |value|, or 0 if no such constant exists in the module. |int_type_id| must exist in the module and must correspond to an integer type id.

Reply:
Done